[2025-01-22T15:08:42.153+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:08:42.155+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:08:42.158+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:08:42.158+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:08:42.220+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:08:42.209+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:08:42.223+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:08:42.252+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.109 seconds
[2025-01-22T15:09:13.115+0000] {processor.py:186} INFO - Started process (PID=124) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:13.116+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:09:13.119+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:09:13.118+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:13.152+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:09:13.143+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:09:13.155+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:13.185+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.076 seconds
[2025-01-22T15:09:44.127+0000] {processor.py:186} INFO - Started process (PID=189) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:44.128+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:09:44.131+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:09:44.131+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:44.158+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:09:44.151+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:09:44.160+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:09:44.179+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.058 seconds
[2025-01-22T15:10:14.788+0000] {processor.py:186} INFO - Started process (PID=254) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:14.790+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:10:14.792+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:10:14.792+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:14.820+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:10:14.813+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:10:14.822+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:14.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.059 seconds
[2025-01-22T15:10:44.947+0000] {processor.py:186} INFO - Started process (PID=319) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:44.948+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:10:44.953+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:10:44.952+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:44.999+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:10:44.990+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:10:45.002+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:10:45.032+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.093 seconds
[2025-01-22T15:11:15.673+0000] {processor.py:186} INFO - Started process (PID=384) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:15.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:11:15.676+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:11:15.676+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:15.709+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:11:15.700+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:11:15.712+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:15.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.072 seconds
[2025-01-22T15:11:46.011+0000] {processor.py:186} INFO - Started process (PID=448) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:46.012+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:11:46.014+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:11:46.014+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:46.036+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:11:46.030+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:11:46.038+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:11:46.059+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:12:32.859+0000] {processor.py:186} INFO - Started process (PID=60) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:12:32.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:12:32.867+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:12:32.866+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:12:32.900+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:12:32.889+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:12:32.903+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:12:32.933+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.084 seconds
[2025-01-22T15:13:03.240+0000] {processor.py:186} INFO - Started process (PID=124) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:03.241+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:13:03.244+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:13:03.243+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:03.270+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:13:03.262+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:13:03.272+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:03.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.056 seconds
[2025-01-22T15:13:33.746+0000] {processor.py:186} INFO - Started process (PID=188) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:33.747+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:13:33.750+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:13:33.750+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:33.777+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:13:33.770+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:13:33.780+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:13:33.810+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.071 seconds
[2025-01-22T15:14:03.895+0000] {processor.py:186} INFO - Started process (PID=252) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:03.896+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:14:03.899+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:14:03.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:03.923+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:14:03.916+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:14:03.925+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:03.943+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.054 seconds
[2025-01-22T15:14:34.330+0000] {processor.py:186} INFO - Started process (PID=316) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:34.331+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:14:34.333+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:14:34.333+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:34.360+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:14:34.353+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:14:34.362+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:14:34.381+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.058 seconds
[2025-01-22T15:15:04.544+0000] {processor.py:186} INFO - Started process (PID=380) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:04.545+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:15:04.548+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:15:04.547+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:04.572+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:15:04.565+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:15:04.575+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:04.597+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.061 seconds
[2025-01-22T15:15:34.920+0000] {processor.py:186} INFO - Started process (PID=444) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:34.921+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:15:34.923+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:15:34.923+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:34.948+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:15:34.941+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:15:34.951+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:15:34.969+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.054 seconds
[2025-01-22T15:16:05.389+0000] {processor.py:186} INFO - Started process (PID=508) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:05.390+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:16:05.392+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:16:05.392+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:05.416+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:16:05.409+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:16:05.419+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:05.436+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.052 seconds
[2025-01-22T15:16:36.311+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:36.312+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:16:36.314+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:16:36.314+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:36.342+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:16:36.334+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:16:36.344+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:16:36.362+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.057 seconds
[2025-01-22T15:17:06.797+0000] {processor.py:186} INFO - Started process (PID=636) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:06.798+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:17:06.800+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:17:06.800+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:06.824+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:17:06.817+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:17:06.826+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:06.846+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:17:37.202+0000] {processor.py:186} INFO - Started process (PID=700) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:37.203+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:17:37.206+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:17:37.205+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:37.230+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:17:37.222+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:17:37.233+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:17:37.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.057 seconds
[2025-01-22T15:18:07.413+0000] {processor.py:186} INFO - Started process (PID=765) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:07.415+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:18:07.418+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:18:07.417+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:07.448+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:18:07.440+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:18:07.450+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:07.470+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.065 seconds
[2025-01-22T15:18:37.561+0000] {processor.py:186} INFO - Started process (PID=836) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:37.562+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:18:37.564+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:18:37.564+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:37.585+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:18:37.580+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:18:37.588+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:18:37.607+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.054 seconds
[2025-01-22T15:19:08.123+0000] {processor.py:186} INFO - Started process (PID=900) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:08.124+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:19:08.126+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:19:08.126+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:08.151+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:19:08.145+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:19:08.154+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:08.173+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:19:38.955+0000] {processor.py:186} INFO - Started process (PID=964) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:38.957+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:19:38.959+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:19:38.959+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:38.986+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:19:38.979+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:19:38.989+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:19:39.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.061 seconds
[2025-01-22T15:20:09.942+0000] {processor.py:186} INFO - Started process (PID=1028) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:09.944+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:20:09.951+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:20:09.950+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:09.987+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:20:09.977+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:20:09.990+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:10.017+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.083 seconds
[2025-01-22T15:20:40.510+0000] {processor.py:186} INFO - Started process (PID=1092) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:40.511+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:20:40.513+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:20:40.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:40.537+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:20:40.529+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:20:40.540+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:20:40.557+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.053 seconds
[2025-01-22T15:21:11.277+0000] {processor.py:186} INFO - Started process (PID=1156) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:11.278+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:21:11.281+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:21:11.280+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:11.300+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:21:11.294+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:21:11.302+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:11.323+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.054 seconds
[2025-01-22T15:21:41.409+0000] {processor.py:186} INFO - Started process (PID=1220) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:41.410+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:21:41.413+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:21:41.413+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:41.441+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:21:41.433+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:21:41.443+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:21:41.463+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.061 seconds
[2025-01-22T15:22:12.100+0000] {processor.py:186} INFO - Started process (PID=1284) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:12.101+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:22:12.104+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:22:12.103+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:12.129+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:22:12.122+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:22:12.131+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:12.150+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:22:42.254+0000] {processor.py:186} INFO - Started process (PID=1348) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:42.255+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:22:42.257+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:22:42.257+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:42.282+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:22:42.275+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:22:42.284+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:22:42.304+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:23:12.876+0000] {processor.py:186} INFO - Started process (PID=1412) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:12.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:23:12.879+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:23:12.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:12.906+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:23:12.898+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:23:12.908+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:12.929+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.059 seconds
[2025-01-22T15:23:43.473+0000] {processor.py:186} INFO - Started process (PID=1475) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:43.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:23:43.476+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:23:43.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:43.502+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:23:43.494+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:23:43.504+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:23:43.523+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:24:13.801+0000] {processor.py:186} INFO - Started process (PID=1539) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:13.802+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:24:13.805+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:24:13.804+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:13.824+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:24:13.819+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:24:13.827+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:13.846+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.049 seconds
[2025-01-22T15:24:43.908+0000] {processor.py:186} INFO - Started process (PID=1603) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:43.909+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:24:43.912+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:24:43.912+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:43.939+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:24:43.932+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:24:43.941+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:24:43.965+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.063 seconds
[2025-01-22T15:25:14.227+0000] {processor.py:186} INFO - Started process (PID=1668) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:14.228+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:25:14.231+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:25:14.231+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:14.254+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:25:14.247+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:25:14.256+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:14.275+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.055 seconds
[2025-01-22T15:25:45.296+0000] {processor.py:186} INFO - Started process (PID=1732) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:45.297+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:25:45.299+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:25:45.299+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:45.325+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:25:45.318+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:25:45.327+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:25:45.347+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.056 seconds
[2025-01-22T15:26:16.250+0000] {processor.py:186} INFO - Started process (PID=1796) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:16.251+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:26:16.253+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:26:16.253+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:16.276+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:26:16.270+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:26:16.279+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:16.301+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.056 seconds
[2025-01-22T15:26:46.487+0000] {processor.py:186} INFO - Started process (PID=1860) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:46.488+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:26:46.491+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:26:46.491+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:46.520+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:26:46.512+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:26:46.523+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:26:46.549+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.068 seconds
[2025-01-22T15:27:16.709+0000] {processor.py:186} INFO - Started process (PID=1911) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:16.711+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:27:16.713+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:27:16.713+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:16.738+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:27:16.731+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:27:16.741+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:16.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.057 seconds
[2025-01-22T15:27:47.755+0000] {processor.py:186} INFO - Started process (PID=1972) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:47.756+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:27:47.758+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:27:47.758+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:47.785+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:27:47.778+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:27:47.788+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:27:47.810+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.061 seconds
[2025-01-22T15:28:17.911+0000] {processor.py:186} INFO - Started process (PID=2027) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:17.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:28:17.915+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:28:17.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:17.942+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:28:17.935+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:28:17.945+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:17.969+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.063 seconds
[2025-01-22T15:28:48.229+0000] {processor.py:186} INFO - Started process (PID=2091) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:48.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:28:48.232+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:28:48.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:48.254+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:28:48.249+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:28:48.256+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:28:48.275+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.051 seconds
[2025-01-22T15:29:18.593+0000] {processor.py:186} INFO - Started process (PID=2155) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:18.594+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:29:18.596+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:29:18.596+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:18.621+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:29:18.615+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:29:18.623+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:18.644+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.058 seconds
[2025-01-22T15:29:49.113+0000] {processor.py:186} INFO - Started process (PID=2208) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:49.114+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:29:49.116+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:29:49.116+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:49.146+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:29:49.138+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:29:49.149+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:29:49.170+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.062 seconds
[2025-01-22T15:42:02.575+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:02.584+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:42:02.589+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:42:02.588+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:02.951+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:42:02.874+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:42:02.967+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:03.074+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.564 seconds
[2025-01-22T15:42:33.468+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:33.468+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:42:33.470+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:42:33.470+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:33.491+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:42:33.485+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:42:33.493+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:42:33.508+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.045 seconds
[2025-01-22T15:43:03.582+0000] {processor.py:186} INFO - Started process (PID=201) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:43:03.583+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:43:03.585+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:43:03.585+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:43:03.610+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:43:03.604+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:43:03.612+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:43:03.630+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.053 seconds
[2025-01-22T15:46:54.165+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:46:54.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:46:54.169+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:46:54.169+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:46:54.196+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:46:54.187+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:46:54.198+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:46:54.220+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.060 seconds
[2025-01-22T15:47:24.978+0000] {processor.py:186} INFO - Started process (PID=136) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:24.979+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:47:24.982+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:47:24.982+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:25.008+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:47:25.000+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:47:25.010+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:25.036+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.063 seconds
[2025-01-22T15:47:55.191+0000] {processor.py:186} INFO - Started process (PID=200) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:55.191+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:47:55.194+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:47:55.194+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:55.224+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:47:55.217+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:47:55.226+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:47:55.244+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.060 seconds
[2025-01-22T15:48:25.778+0000] {processor.py:186} INFO - Started process (PID=264) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:25.779+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:48:25.781+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:48:25.781+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:25.804+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:48:25.798+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:48:25.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:25.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.050 seconds
[2025-01-22T15:48:56.304+0000] {processor.py:186} INFO - Started process (PID=327) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:56.305+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:48:56.309+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:48:56.308+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:56.334+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:48:56.327+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:48:56.337+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:48:56.357+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.060 seconds
[2025-01-22T15:49:26.399+0000] {processor.py:186} INFO - Started process (PID=391) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:49:26.400+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T15:49:26.401+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:49:26.401+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:49:26.422+0000] {logging_mixin.py:190} INFO - [2025-01-22T15:49:26.416+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline_dag.py", line 4, in <module>
    from main import data_ingestion, data_preprocessing, train, deploy_to_HF
  File "/opt/airflow/dags/main.py", line 1, in <module>
    from datasets import DatasetDict, load_dataset
ModuleNotFoundError: No module named 'datasets'
[2025-01-22T15:49:26.424+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T15:49:26.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 0.047 seconds
[2025-01-22T16:13:15.854+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:13:15.856+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:13:15.858+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:15.858+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:13:16.286+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:16.286+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:13:33.805+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:13:33.815+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:13:33.930+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.929+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:fine_tune_pipeline
[2025-01-22T16:13:33.942+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.941+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:fine_tune_pipeline
[2025-01-22T16:13:33.950+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.950+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:fine_tune_pipeline
[2025-01-22T16:13:33.958+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.958+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:fine_tune_pipeline
[2025-01-22T16:13:33.967+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.966+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:fine_tune_pipeline
[2025-01-22T16:13:33.975+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.974+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:fine_tune_pipeline
[2025-01-22T16:13:33.981+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.981+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:fine_tune_pipeline
[2025-01-22T16:13:33.982+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.982+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:13:33.995+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.994+0000] {dag.py:3262} INFO - Creating ORM DAG for fine_tune_pipeline
[2025-01-22T16:13:33.996+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:13:33.996+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:13:34.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 18.173 seconds
[2025-01-22T16:14:04.567+0000] {processor.py:186} INFO - Started process (PID=198) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:14:04.568+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:14:04.571+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:04.570+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:14:04.866+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:04.865+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:14:17.794+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:14:17.807+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:14:17.833+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:17.832+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:14:17.852+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:17.852+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:14:17.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.317 seconds
[2025-01-22T16:14:48.900+0000] {processor.py:186} INFO - Started process (PID=281) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:14:48.900+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:14:48.903+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:48.903+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:14:49.175+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:14:49.175+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:15:00.283+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:15:00.289+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:15:00.307+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:00.307+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:15:00.320+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:00.320+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:15:00.338+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.444 seconds
[2025-01-22T16:15:30.587+0000] {processor.py:186} INFO - Started process (PID=381) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:15:30.588+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:15:30.590+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:30.590+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:15:30.833+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:30.833+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:15:41.222+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:15:41.228+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:15:41.247+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:41.246+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:15:41.261+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:15:41.261+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:15:41.283+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.702 seconds
[2025-01-22T16:16:12.089+0000] {processor.py:186} INFO - Started process (PID=486) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:16:12.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:16:12.092+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:12.092+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:16:12.320+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:12.320+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:16:20.944+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:16:20.949+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:16:20.967+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:20.965+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:16:21.210+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:21.210+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:16:21.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 9.143 seconds
[2025-01-22T16:16:51.389+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:16:51.390+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:16:51.392+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:51.392+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:16:51.602+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:16:51.602+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:17:00.703+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:17:00.708+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:17:00.725+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:00.725+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:17:00.737+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:00.737+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:17:00.754+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 9.369 seconds
[2025-01-22T16:17:30.859+0000] {processor.py:186} INFO - Started process (PID=656) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:17:30.860+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:17:30.862+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:30.862+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:17:31.098+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:31.098+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:17:46.394+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:17:46.408+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:17:46.431+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:46.430+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:17:46.443+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:17:46.443+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:17:46.462+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 15.609 seconds
[2025-01-22T16:18:16.750+0000] {processor.py:186} INFO - Started process (PID=776) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:18:16.751+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:18:16.754+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:16.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:18:17.061+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:17.061+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:18:27.868+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:18:27.873+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:18:27.890+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:27.890+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:18:27.905+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:27.904+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:18:27.927+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.183 seconds
[2025-01-22T16:18:58.078+0000] {processor.py:186} INFO - Started process (PID=861) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:18:58.079+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:18:58.081+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:58.081+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:18:58.319+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:18:58.318+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:19:08.562+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:19:08.567+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:19:08.587+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:08.586+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:19:08.601+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:08.600+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:19:08.620+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.548 seconds
[2025-01-22T16:19:38.773+0000] {processor.py:186} INFO - Started process (PID=948) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:19:38.774+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:19:38.776+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:38.776+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:19:39.051+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:39.051+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:19:51.403+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:19:51.407+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:19:51.423+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:51.423+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:19:51.435+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:19:51.435+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:19:51.457+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.691 seconds
[2025-01-22T16:20:21.553+0000] {processor.py:186} INFO - Started process (PID=1061) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:20:21.554+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:20:21.556+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:20:21.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:20:21.837+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:20:21.837+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:20:31.768+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:20:31.773+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:20:31.791+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:20:31.789+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:20:31.802+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:20:31.802+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:20:31.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.278 seconds
[2025-01-22T16:21:01.978+0000] {processor.py:186} INFO - Started process (PID=1147) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:21:01.979+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:21:01.982+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:01.982+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:21:02.288+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:02.288+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:21:16.746+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:21:16.765+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:21:16.796+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:16.795+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:21:16.817+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:16.817+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:21:16.848+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.875 seconds
[2025-01-22T16:21:47.307+0000] {processor.py:186} INFO - Started process (PID=1251) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:21:47.308+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:21:47.313+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:47.312+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:21:47.709+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:21:47.708+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:22:01.501+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:22:01.507+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:22:01.528+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:01.527+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:22:01.544+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:01.544+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:22:01.568+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.270 seconds
[2025-01-22T16:22:32.287+0000] {processor.py:186} INFO - Started process (PID=1359) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:22:32.288+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:22:32.290+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:32.290+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:22:32.586+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:32.586+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:22:44.036+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:22:44.045+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:22:44.064+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:44.063+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:22:44.077+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:22:44.077+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:22:44.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.819 seconds
[2025-01-22T16:23:14.441+0000] {processor.py:186} INFO - Started process (PID=1445) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:23:14.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:23:14.444+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:14.444+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:23:14.721+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:14.721+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:23:26.498+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:23:26.503+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:23:26.522+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:26.522+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:23:26.537+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:26.536+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:23:26.561+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.125 seconds
[2025-01-22T16:23:56.861+0000] {processor.py:186} INFO - Started process (PID=1558) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:23:56.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:23:56.864+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:56.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:23:57.155+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:23:57.155+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:24:10.676+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:24:10.682+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:24:10.698+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:10.697+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:24:10.710+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:10.710+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:24:10.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.872 seconds
[2025-01-22T16:24:41.435+0000] {processor.py:186} INFO - Started process (PID=1648) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:24:41.436+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:24:41.439+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:41.439+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:24:41.758+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:41.757+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:24:54.679+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:24:54.686+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:24:54.704+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:54.703+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:24:54.718+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:24:54.717+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:24:54.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.311 seconds
[2025-01-22T16:25:24.844+0000] {processor.py:186} INFO - Started process (PID=1742) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:25:24.845+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:25:24.848+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:25:24.847+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:25:25.158+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:25:25.157+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:25:37.518+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:25:37.524+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:25:37.543+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:25:37.542+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:25:37.555+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:25:37.555+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:25:37.574+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.740 seconds
[2025-01-22T16:26:07.894+0000] {processor.py:186} INFO - Started process (PID=1856) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:26:07.895+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:26:07.898+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:07.897+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:26:08.142+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:08.142+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:26:19.325+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:26:19.330+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:26:19.350+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:19.348+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:26:19.365+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:19.365+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:26:19.388+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.499 seconds
[2025-01-22T16:26:49.535+0000] {processor.py:186} INFO - Started process (PID=1951) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:26:49.536+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:26:49.539+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:49.538+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:26:49.854+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:26:49.853+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:27:04.248+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:27:04.258+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:27:04.279+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:04.278+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:27:04.295+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:04.295+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:27:04.324+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.797 seconds
[2025-01-22T16:27:34.591+0000] {processor.py:186} INFO - Started process (PID=2029) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:27:34.592+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:27:34.594+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:34.594+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:27:34.909+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:34.909+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:27:46.946+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:27:46.950+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:27:46.969+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:46.969+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:27:46.983+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:27:46.982+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:27:47.002+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.416 seconds
[2025-01-22T16:28:17.238+0000] {processor.py:186} INFO - Started process (PID=2146) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:28:17.239+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:28:17.241+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:17.240+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:28:17.520+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:17.519+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:28:27.027+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:28:27.032+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:28:27.058+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:27.057+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:28:27.070+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:27.069+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:28:27.087+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 9.854 seconds
[2025-01-22T16:28:57.427+0000] {processor.py:186} INFO - Started process (PID=2238) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:28:57.428+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:28:57.429+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:57.429+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:28:57.650+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:28:57.650+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:29:06.404+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:29:06.410+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:29:06.428+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:06.427+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:29:06.441+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:06.440+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:29:06.458+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 9.038 seconds
[2025-01-22T16:29:36.950+0000] {processor.py:186} INFO - Started process (PID=2315) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:29:36.951+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:29:36.954+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:36.953+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:29:37.226+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:37.226+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:29:48.374+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:29:48.378+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:29:48.395+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:48.394+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:29:48.406+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:29:48.406+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:29:48.422+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.478 seconds
[2025-01-22T16:30:19.061+0000] {processor.py:186} INFO - Started process (PID=2433) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:30:19.062+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:30:19.065+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:19.064+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:30:19.392+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:19.391+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:30:29.130+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:30:29.139+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:30:29.191+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:29.190+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:30:29.204+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:29.204+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:30:29.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.173 seconds
[2025-01-22T16:30:59.339+0000] {processor.py:186} INFO - Started process (PID=2524) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:30:59.340+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:30:59.342+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:59.341+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:30:59.574+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:30:59.574+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:31:11.232+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:31:11.237+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:31:11.255+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:11.254+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:31:11.268+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:11.268+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:31:11.286+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.954 seconds
[2025-01-22T16:31:41.513+0000] {processor.py:186} INFO - Started process (PID=2603) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:31:41.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:31:41.516+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:41.515+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:31:41.771+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:41.770+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:31:52.998+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:31:53.003+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:31:53.019+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:53.019+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:31:53.031+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:31:53.031+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:31:53.047+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.539 seconds
[2025-01-22T16:32:23.213+0000] {processor.py:186} INFO - Started process (PID=2733) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:32:23.214+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:32:23.216+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:32:23.216+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:32:23.468+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:32:23.467+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:32:33.175+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:32:33.182+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:32:33.215+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:32:33.215+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:32:33.228+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:32:33.227+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:32:33.250+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.048 seconds
[2025-01-22T16:33:04.218+0000] {processor.py:186} INFO - Started process (PID=2809) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:04.219+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:33:04.221+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:04.220+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:04.468+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:04.468+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:33:12.523+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:33:12.530+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:12.545+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:12.545+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:33:12.557+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:12.557+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:33:12.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 8.364 seconds
[2025-01-22T16:33:43.533+0000] {processor.py:186} INFO - Started process (PID=2884) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:43.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:33:43.538+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:43.537+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:43.734+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:43.733+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:33:54.432+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:33:54.437+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:33:54.459+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:54.458+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:33:54.475+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:33:54.474+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:33:54.501+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.973 seconds
[2025-01-22T16:34:25.500+0000] {processor.py:186} INFO - Started process (PID=3006) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:34:25.501+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:34:25.503+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:34:25.503+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:34:25.975+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:34:25.975+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:34:37.766+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:34:37.772+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:34:37.791+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:34:37.791+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:34:37.803+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:34:37.803+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:34:37.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.334 seconds
[2025-01-22T16:35:07.920+0000] {processor.py:186} INFO - Started process (PID=3094) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:35:07.921+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:35:07.923+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:07.923+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:35:08.323+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:08.323+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:35:17.724+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:35:17.744+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:35:17.761+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:17.760+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:35:17.773+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:17.773+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:35:17.789+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 9.874 seconds
[2025-01-22T16:35:48.652+0000] {processor.py:186} INFO - Started process (PID=3174) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:35:48.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:35:48.655+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:48.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:35:49.080+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:35:49.080+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:36:00.761+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:36:00.766+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:36:00.808+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:00.808+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:36:00.822+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:00.821+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:36:00.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.193 seconds
[2025-01-22T16:36:31.668+0000] {processor.py:186} INFO - Started process (PID=3304) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:36:31.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:36:31.671+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:31.671+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:36:32.111+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:32.111+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:36:41.911+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:36:41.919+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:36:41.965+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:41.964+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:36:41.978+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:36:41.978+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:36:41.998+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.333 seconds
[2025-01-22T16:37:12.516+0000] {processor.py:186} INFO - Started process (PID=3379) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:37:12.517+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:37:12.519+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:12.518+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:37:12.767+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:12.767+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:37:24.198+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:37:24.210+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:37:24.327+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:24.326+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:37:24.345+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:24.344+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:37:24.382+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.871 seconds
[2025-01-22T16:37:54.965+0000] {processor.py:186} INFO - Started process (PID=3468) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:37:54.966+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:37:54.968+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:54.967+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:37:55.271+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:37:55.271+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:38:08.612+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:38:08.617+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:38:08.642+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:08.642+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:38:08.654+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:08.654+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:38:08.671+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.713 seconds
[2025-01-22T16:38:39.286+0000] {processor.py:186} INFO - Started process (PID=3590) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:38:39.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:38:39.289+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:39.289+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:38:39.511+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:39.511+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:38:49.386+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:38:49.391+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:38:49.407+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:49.407+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:38:49.419+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:38:49.419+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:38:49.437+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.155 seconds
[2025-01-22T16:39:20.252+0000] {processor.py:186} INFO - Started process (PID=3666) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:39:20.253+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:39:20.255+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:39:20.255+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:39:20.542+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:39:20.542+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:39:30.806+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:39:30.811+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:39:30.829+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:39:30.829+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:39:30.843+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:39:30.843+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:39:30.867+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.620 seconds
[2025-01-22T16:40:01.474+0000] {processor.py:186} INFO - Started process (PID=3757) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:01.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:40:01.476+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:01.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:01.723+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:01.723+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:40:13.312+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:40:13.317+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:13.341+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:13.340+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:40:13.360+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:13.360+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:40:13.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.917 seconds
[2025-01-22T16:40:43.952+0000] {processor.py:186} INFO - Started process (PID=3875) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:43.953+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:40:43.955+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:43.955+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:44.185+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:44.185+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:40:56.923+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:40:56.928+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:40:56.944+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:56.944+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:40:56.956+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:40:56.956+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:40:56.972+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.025 seconds
[2025-01-22T16:41:27.071+0000] {processor.py:186} INFO - Started process (PID=3964) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:41:27.072+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:41:27.074+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:41:27.074+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:41:27.362+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:41:27.362+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:41:39.419+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:41:39.424+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:41:39.443+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:41:39.442+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:41:39.455+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:41:39.455+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:41:39.472+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.408 seconds
[2025-01-22T16:42:10.458+0000] {processor.py:186} INFO - Started process (PID=4071) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:42:10.459+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:42:10.461+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:10.460+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:42:10.717+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:10.717+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:42:21.832+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:42:21.837+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:42:21.853+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:21.853+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:42:21.867+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:21.867+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:42:21.883+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.430 seconds
[2025-01-22T16:42:52.793+0000] {processor.py:186} INFO - Started process (PID=4162) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:42:52.794+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:42:52.796+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:52.795+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:42:53.037+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:42:53.037+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:43:03.710+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:43:03.716+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:43:03.736+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:03.736+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:43:03.748+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:03.748+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:43:03.765+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.980 seconds
[2025-01-22T16:43:33.956+0000] {processor.py:186} INFO - Started process (PID=4252) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:43:33.957+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:43:33.961+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:33.959+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:43:34.210+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:34.210+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:43:46.325+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:43:46.336+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:43:46.357+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:46.356+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:43:46.374+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:43:46.373+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:43:46.402+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.453 seconds
[2025-01-22T16:44:16.638+0000] {processor.py:186} INFO - Started process (PID=4365) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:44:16.639+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:44:16.642+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:44:16.641+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:44:16.936+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:44:16.936+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:44:30.588+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:44:30.593+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:44:30.611+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:44:30.610+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:44:30.625+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:44:30.625+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:44:30.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.014 seconds
[2025-01-22T16:45:00.748+0000] {processor.py:186} INFO - Started process (PID=4459) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:00.749+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:45:00.751+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:00.751+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:01.014+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:01.013+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:45:12.251+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:45:12.261+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:12.307+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:12.307+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:45:12.321+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:12.321+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:45:12.343+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.603 seconds
[2025-01-22T16:45:43.301+0000] {processor.py:186} INFO - Started process (PID=4538) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:43.301+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:45:43.303+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:43.303+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:43.544+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:43.544+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:45:54.735+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:45:54.741+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:45:54.757+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:54.757+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:45:54.769+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:45:54.769+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:45:54.786+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.491 seconds
[2025-01-22T16:46:25.267+0000] {processor.py:186} INFO - Started process (PID=4657) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:46:25.268+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:46:25.270+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:46:25.270+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:46:25.539+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:46:25.539+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:46:35.399+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:46:35.403+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:46:35.420+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:46:35.419+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:46:35.433+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:46:35.433+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:46:35.450+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.193 seconds
[2025-01-22T16:47:06.335+0000] {processor.py:186} INFO - Started process (PID=4748) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:47:06.336+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:47:06.338+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:06.338+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:47:06.565+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:06.565+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:47:18.888+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:47:18.900+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:47:18.923+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:18.922+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:47:18.947+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:18.947+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:47:18.973+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.643 seconds
[2025-01-22T16:47:49.584+0000] {processor.py:186} INFO - Started process (PID=4823) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:47:49.585+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:47:49.588+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:49.587+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:47:49.929+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:47:49.929+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:48:03.175+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:48:03.180+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:48:03.199+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:03.197+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:48:03.212+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:03.212+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:48:03.232+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.657 seconds
[2025-01-22T16:48:33.427+0000] {processor.py:186} INFO - Started process (PID=4955) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:48:33.428+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:48:33.430+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:33.430+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:48:33.689+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:33.689+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:48:44.273+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:48:44.284+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:48:44.346+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:44.346+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:48:44.359+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:48:44.358+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:48:44.377+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.954 seconds
[2025-01-22T16:49:14.503+0000] {processor.py:186} INFO - Started process (PID=5033) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:49:14.504+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:49:14.506+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:14.505+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:49:14.733+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:14.733+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:49:25.383+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:49:25.388+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:49:25.406+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:25.405+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:49:25.418+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:25.418+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:49:25.437+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.939 seconds
[2025-01-22T16:49:56.291+0000] {processor.py:186} INFO - Started process (PID=5121) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:49:56.292+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:49:56.295+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:56.294+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:49:56.558+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:49:56.558+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:50:08.626+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:50:08.631+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:50:08.647+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:08.646+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:50:08.659+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:08.659+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:50:08.674+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.393 seconds
[2025-01-22T16:50:39.566+0000] {processor.py:186} INFO - Started process (PID=5243) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:50:39.567+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:50:39.569+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:39.569+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:50:39.818+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:39.817+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:50:51.610+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:50:51.623+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:50:51.646+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:51.645+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:50:51.659+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:50:51.658+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:50:51.677+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.116 seconds
[2025-01-22T16:51:21.806+0000] {processor.py:186} INFO - Started process (PID=5317) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:51:21.807+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:51:21.810+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:51:21.810+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:51:22.104+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:51:22.104+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:51:34.835+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:51:34.839+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:51:34.855+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:51:34.854+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:51:34.868+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:51:34.868+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:51:34.885+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.089 seconds
[2025-01-22T16:52:05.701+0000] {processor.py:186} INFO - Started process (PID=5450) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:05.702+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:52:05.704+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:05.704+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:05.924+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:05.924+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:52:16.584+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:52:16.592+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:16.619+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:16.619+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:52:16.631+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:16.631+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:52:16.651+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.954 seconds
[2025-01-22T16:52:47.148+0000] {processor.py:186} INFO - Started process (PID=5529) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:47.148+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:52:47.150+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:47.150+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:47.375+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:47.375+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:52:57.332+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:52:57.336+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:52:57.356+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:57.355+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:52:57.368+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:52:57.368+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:52:57.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 10.240 seconds
[2025-01-22T16:53:27.942+0000] {processor.py:186} INFO - Started process (PID=5613) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:53:27.943+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:53:27.946+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:53:27.946+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:53:28.205+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:53:28.205+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:53:39.603+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:53:39.609+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:53:39.628+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:53:39.627+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:53:39.641+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:53:39.641+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:53:39.662+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.737 seconds
[2025-01-22T16:54:09.926+0000] {processor.py:186} INFO - Started process (PID=5735) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:54:09.927+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:54:09.930+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:09.930+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:54:10.191+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:10.191+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:54:22.858+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:54:22.871+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:54:22.897+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:22.896+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:54:22.909+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:22.909+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:54:22.931+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.011 seconds
[2025-01-22T16:54:53.222+0000] {processor.py:186} INFO - Started process (PID=5814) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:54:53.223+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:54:53.226+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:53.225+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:54:53.506+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:54:53.506+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:55:05.233+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:55:05.239+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:55:05.255+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:05.255+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:55:05.267+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:05.267+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:55:05.282+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.068 seconds
[2025-01-22T16:55:35.904+0000] {processor.py:186} INFO - Started process (PID=5910) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:55:35.905+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:55:35.907+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:35.907+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:55:36.156+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:36.155+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:55:48.161+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:55:48.167+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:55:48.183+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:48.182+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:55:48.195+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:55:48.195+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:55:48.211+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.311 seconds
[2025-01-22T16:56:18.854+0000] {processor.py:186} INFO - Started process (PID=6025) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:56:18.855+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:56:18.857+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:56:18.856+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:56:19.080+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:56:19.079+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:56:30.660+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T16:56:30.665+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:56:30.683+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:56:30.682+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T16:56:30.696+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:56:30.696+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T16:56:30.716+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.866 seconds
[2025-01-22T16:57:00.898+0000] {processor.py:186} INFO - Started process (PID=6117) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:57:00.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T16:57:00.900+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:57:00.900+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T16:57:01.133+0000] {logging_mixin.py:190} INFO - [2025-01-22T16:57:01.132+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T16:57:17.953+0000] {logging_mixin.py:190} WARNING - wandb: W&B API key is configured. Use `wandb login --relogin` to force relogin
[2025-01-22T16:57:19.010+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:02:35.826+0000] {processor.py:186} INFO - Started process (PID=6200) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:02:35.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:02:35.828+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:02:35.828+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:02:36.084+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:02:36.083+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:02:50.533+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:02:50.537+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:02:50.674+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:02:50.673+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:02:50.684+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:02:50.684+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:02:50.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.904 seconds
[2025-01-22T17:03:21.267+0000] {processor.py:186} INFO - Started process (PID=6336) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:03:21.267+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:03:21.268+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:03:21.268+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:03:21.502+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:03:21.502+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:03:32.377+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:03:32.386+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:03:32.428+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:03:32.427+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:03:32.686+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:03:32.685+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:03:32.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.444 seconds
[2025-01-22T17:04:02.838+0000] {processor.py:186} INFO - Started process (PID=6410) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:02.839+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:04:02.840+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:02.840+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:03.062+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:03.062+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:04:14.264+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:04:14.269+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:14.289+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:14.286+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:04:14.302+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:14.302+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:04:14.322+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.489 seconds
[2025-01-22T17:04:44.528+0000] {processor.py:186} INFO - Started process (PID=6516) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:44.529+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:04:44.530+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:44.530+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:44.758+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:44.757+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:04:57.087+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:04:57.092+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:04:57.111+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:57.110+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:04:57.123+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:04:57.123+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:04:57.141+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.617 seconds
[2025-01-22T17:05:27.901+0000] {processor.py:186} INFO - Started process (PID=6621) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:05:27.902+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:05:27.904+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:05:27.903+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:05:28.134+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:05:28.133+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:05:40.590+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:05:40.595+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:05:40.617+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:05:40.615+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:05:41.057+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:05:41.057+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:05:41.077+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.190 seconds
[2025-01-22T17:06:11.249+0000] {processor.py:186} INFO - Started process (PID=6708) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:06:11.250+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:06:11.251+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:11.250+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:06:11.538+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:11.537+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:06:25.024+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:06:25.028+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:06:25.046+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:25.044+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:06:25.300+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:25.300+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:06:25.319+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.075 seconds
[2025-01-22T17:06:55.655+0000] {processor.py:186} INFO - Started process (PID=6832) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:06:55.655+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:06:55.656+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:55.656+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:06:55.882+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:06:55.882+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:07:07.294+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:07:07.302+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:07:07.351+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:07.350+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:07:07.364+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:07.363+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:07:07.385+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.737 seconds
[2025-01-22T17:07:38.166+0000] {processor.py:186} INFO - Started process (PID=6906) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:07:38.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:07:38.169+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:38.168+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:07:38.437+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:38.437+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:07:49.712+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:07:49.721+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:07:49.752+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:49.751+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:07:49.770+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:07:49.770+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:07:49.789+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 11.630 seconds
[2025-01-22T17:08:20.169+0000] {processor.py:186} INFO - Started process (PID=7005) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:08:20.170+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:08:20.171+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:08:20.171+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:08:20.464+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:08:20.464+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:08:32.938+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:08:32.944+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:08:32.969+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:08:32.968+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:08:32.982+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:08:32.981+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:08:33.004+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.842 seconds
[2025-01-22T17:09:03.139+0000] {processor.py:186} INFO - Started process (PID=7117) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:03.140+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:09:03.141+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:03.141+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:03.367+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:03.367+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:09:16.062+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:09:16.067+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:16.084+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:16.084+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:09:16.096+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:16.096+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:09:16.112+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.979 seconds
[2025-01-22T17:09:46.564+0000] {processor.py:186} INFO - Started process (PID=7210) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:46.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:09:46.566+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:46.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:46.871+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:46.871+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:09:59.908+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:09:59.913+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:09:59.933+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:59.933+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:09:59.947+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:09:59.947+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:09:59.984+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.427 seconds
[2025-01-22T17:10:30.078+0000] {processor.py:186} INFO - Started process (PID=7320) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:10:30.080+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:10:30.082+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:10:30.081+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:10:30.434+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:10:30.433+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:10:44.358+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:10:44.363+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:10:44.381+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:10:44.380+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:10:44.394+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:10:44.394+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:10:44.414+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.342 seconds
[2025-01-22T17:11:14.607+0000] {processor.py:186} INFO - Started process (PID=7412) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:11:14.608+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:11:14.609+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:14.608+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:11:14.878+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:14.878+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:11:26.942+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:11:26.946+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:11:26.965+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:26.964+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:11:26.977+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:26.977+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:11:26.994+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.392 seconds
[2025-01-22T17:11:57.779+0000] {processor.py:186} INFO - Started process (PID=7508) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:11:57.781+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:11:57.782+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:57.782+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:11:58.068+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:11:58.068+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:12:14.766+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:12:14.778+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:12:14.854+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:12:14.853+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:12:14.871+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:12:14.871+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:12:14.900+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 17.129 seconds
[2025-01-22T17:12:45.615+0000] {processor.py:186} INFO - Started process (PID=7621) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:12:45.616+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:12:45.618+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:12:45.618+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:12:45.887+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:12:45.887+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:13:01.564+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:13:01.568+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:13:01.586+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:01.584+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:13:01.598+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:01.598+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:13:01.616+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 16.009 seconds
[2025-01-22T17:13:32.067+0000] {processor.py:186} INFO - Started process (PID=7723) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:13:32.067+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:13:32.068+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:32.068+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:13:32.308+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:32.308+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:13:44.946+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:13:44.951+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:13:44.967+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:44.967+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:13:44.979+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:13:44.979+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:13:44.995+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.937 seconds
[2025-01-22T17:14:15.106+0000] {processor.py:186} INFO - Started process (PID=7834) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:14:15.107+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:14:15.108+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:15.108+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:14:15.367+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:15.367+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:14:28.231+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:14:28.235+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:14:28.253+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:28.253+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:14:28.267+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:28.266+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:14:28.290+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.189 seconds
[2025-01-22T17:14:59.091+0000] {processor.py:186} INFO - Started process (PID=7931) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:14:59.093+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:14:59.094+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:59.094+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:14:59.349+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:14:59.348+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:15:11.129+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:15:11.134+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:15:11.154+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:11.153+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:15:11.169+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:11.169+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:15:11.196+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 12.117 seconds
[2025-01-22T17:15:41.298+0000] {processor.py:186} INFO - Started process (PID=8012) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:15:41.299+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:15:41.300+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:41.300+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:15:41.580+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:41.580+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:15:56.791+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:15:56.799+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:15:56.819+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:56.818+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:15:56.831+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:15:56.831+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:15:56.854+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 15.561 seconds
[2025-01-22T17:16:27.092+0000] {processor.py:186} INFO - Started process (PID=8135) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:16:27.093+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:16:27.094+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:16:27.094+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:16:27.368+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:16:27.367+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:16:40.356+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:16:40.361+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:16:40.380+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:16:40.379+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:16:40.393+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:16:40.392+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:16:40.412+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 13.327 seconds
[2025-01-22T17:17:10.783+0000] {processor.py:186} INFO - Started process (PID=8221) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:17:10.785+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:17:10.786+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:10.786+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:17:11.179+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:11.179+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:17:24.951+0000] {logging_mixin.py:190} WARNING - wandb: Currently logged in as: 0132114 (0132114-uow-malaysia). Use `wandb login --relogin` to force relogin
[2025-01-22T17:17:24.957+0000] {processor.py:925} INFO - DAG(s) 'fine_tune_pipeline' retrieved from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:17:24.975+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:24.975+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-22T17:17:24.988+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:24.988+0000] {dag.py:4180} INFO - Setting next_dagrun for fine_tune_pipeline to None, run_after=None
[2025-01-22T17:17:25.007+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/pipeline_dag.py took 14.234 seconds
[2025-01-22T17:17:56.500+0000] {processor.py:186} INFO - Started process (PID=8342) to work on /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:17:56.515+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/pipeline_dag.py for tasks to queue
[2025-01-22T17:17:56.517+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:56.517+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline_dag.py
[2025-01-22T17:17:57.463+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:17:57.462+0000] {config.py:54} INFO - PyTorch version 2.5.1 available.
[2025-01-22T17:18:22.487+0000] {logging_mixin.py:190} INFO - [2025-01-22T17:18:22.487+0000] {process_utils.py:266} INFO - Waiting up to 5 seconds for processes to exit...
